tests/unit/ui/components/TestLoadingText.js

```diff
@@ -0,0 +1,24 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import LoadingText from 'ui/components/LoadingText';
+
+
+describe('<LoadingText />', () => {
+  const render = (props = {}) => shallow(<LoadingText {...props} />);
+
+  it('renders LoadingText element with className', () => {
+    const root = render();
+    expect(root).toHaveClassName('LoadingText');
+  });
+
+  it('lets you set a fixed width', () => {
+    const root = render({ width: 55 });
+    expect(root.prop('style')).toMatchObject({ width: '55%' });
```

Comment:
This can be `expect(root).toHaveProp('style', { width: '55%' });` because I don't think we expect the component to be setting style props other than width.